remotemonitoring/single-vm/scripts/simulate.sh

```diff
@@ -1,19 +1,13 @@
 #!/bin/bash -e
 
-# TODO: the script doesn't work due to Auth
-#       https://github.com/Azure/device-simulation-dotnet/issues/66
-
 cd /app
 
-echo "Script temporarily disabled, see https://github.com/Azure/device-simulation-dotnet/issues/66"
-exit -1
-
 echo "Starting simulation..."
-ISUP=$(curl -sk https://localhost/devicesimulation/v1/status | grep "Alive" | wc -l)
+ISUP=$(docker exec -it app_devicesimulation_1 curl -sk http://localhost:9003/v1/status | grep "Alive" | wc -l)
 while [[ "$ISUP" == "0" ]]; do
   echo "Waiting for simulation service to be available..."
   sleep 4
-  ISUP=$(curl -sk https://localhost/devicesimulation/v1/status | grep "Alive" | wc -l)
+  ISUP=$(docker exec -it app_devicesimulation_1 curl -sk http://localhost:9003/v1/status |grep "Alive" | wc -l)
```

Comment:
this is a bit brittle; i.e. it's dependent on the response string Alive being in there.  Probably fine but is there something else we should key off of?  e.g. should we just use a 200/OK?

Reply:
sure, there are things we can improve, status code should do but I don't want to change that now. I'll track it as a separate task